src/coverlet.core/Instrumentation/Instrumenter.cs

```diff
@@ -43,6 +43,14 @@ public Instrumenter(string module, string identifier, string[] excludeFilters, s
             _excludedAttributes = excludedAttributes;
         }
 
+        private bool IsCoreLibrary
```

Comment:
```suggestion
        private bool IsCoreLibrary { get; } = Path.GetFileNameWithoutExtension(_module) == "System.Private.CoreLib";
```

Reply:
Won't that throw a `NullReferenceException` at the beginning of the constructor?